docs/api/embed.rst

```diff
@@ -0,0 +1,135 @@
+         }
+         response = requests.get(URL, params=params)
+         print(response.json())
+
+         # Or using the URL
+         params = {
+            'url': 'https://docs.readthedocs.io/en/latest/badges.html#style',
+         }
+         response = requests.get(URL, params=params)
+         print(response.json())
+
+   **Example response**:
+
+   .. sourcecode:: json
+
+      {
+         "content": [
+            "<div class=\"section\" id=\"style\">\n<h2>Style<a class=\"headerlink\" href=\"https://docs.readthedocs.io/en/latest/badges.html#style\" title=\"Permalink to this headline\">¶</a></h2>\n<p>Now you can pass the <code class=\"docutils literal notranslate\"><span class=\"pre\">style</span></code> GET argument,\nto get custom styled badges same as you would for shields.io.\nIf no argument is passed, <code class=\"docutils literal notranslate\"><span class=\"pre\">flat</span></code> is used as default.</p>\n<table class=\"docutils align-default\">\n<colgrouisp>\n<col style=\"width: 42%\">\n<col style=\"width: 58%\">\n</colgroup>\n<thead>\n<tr class=\"row-odd\"><th class=\"head\"><p>STYLE</p></th>\n<th class=\"head\"><p>BADGE</p></th>\n</tr>\n</thead>\n<tbody>\n<tr class=\"row-even\"><td><p>flat</p></td>\n<td><p><img alt=\"Flat Badge\" src=\"https://readthedocs.org/projects/pip/badge/?version=latest&amp;style=flat\"></p></td>\n</tr>\n<tr class=\"row-odd\"><td><p>flat-square</p></td>\n<td><p><img alt=\"Flat-Square Badge\" src=\"https://readthedocs.org/projects/pip/badge/?version=latest&amp;style=flat-square\"></p></td>\n</tr>\n<tr class=\"row-even\"><td><p>for-the-badge</p></td>\n<td><p><img alt=\"Badge\" src=\"https://readthedocs.org/projects/pip/badge/?version=latest&amp;style=for-the-badge\"></p></td>\n</tr>\n<tr class=\"row-odd\"><td><p>plastic</p></td>\n<td><p><img alt=\"Plastic Badge\" src=\"https://readthedocs.org/projects/pip/badge/?version=latest&amp;style=plastic\"></p></td>\n</tr>\n<tr class=\"row-even\"><td><p>social</p></td>\n<td><p><img alt=\"Social Badge\" src=\"https://readthedocs.org/projects/pip/badge/?version=latest&amp;style=social\"></p></td>\n</tr>\n</tbody>\n</table>\n</div>"
+         ],
+         "headers": [
+            {
+                  "Badges": "#"
+            },
+            {
+                  "Status Badges": "#status-badges"
+            },
+            {
+                  "Style": "#style"
+            },
+            {
+                  "Project Pages": "#project-pages"
+            }
+         ],
+         "url": "https://docs.readthedocs.io/en/latest/badges.html",
+         "meta": {
+            "project": "docs",
+            "version": "latest",
+            "doc": "badges",
+            "section": "style"
+         }
+      }
+
+Authentication and authorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using :ref:`private versions <versions:privacy levels>`,
+users will only be allowed to get the embed content from projects they have permissions over.
```

Comment:
This seems like it's maybe changing with CORS changes

Reply:
Nope, this doesn't makes reference to external site, but to requests from the same site.